flake.nix

```diff
@@ -0,0 +1,45 @@
+{
+    description = "Tool to simplify secure shell connections over AWS simple systems manager.";
+
+    inputs = {
+      nixpkgs.url = "github:NixOS/nixpkgs?ref=nixpkgs-unstable";
+      utils.url = "github:numtide/flake-utils";
+    };
+    outputs = { self, nixpkgs, utils }: {
+      devShell = self.defaultPackage;
+      defaultPackage.x86_64-darwin =
+        with import nixpkgs { system = "x86_64-darwin"; };
+        let
+          awssh = pkgs.python310Packages.buildPythonPackage rec {
+            pname = "awssh";
+            version = "1.1.0";
```

Comment:
It would also be great if our `bump_version.sh` script were smart enough to bump the version numbers in this file as well.  We do something similar in the [`bump_version.sh` in `cisagov/postfix-docker`](https://github.com/cisagov/postfix-docker/blob/develop/bump_version.sh).

Reply:
I'm not sure if I understand. This would change on every release though and is currently being built for the current release. This is not really a packaged program

Reply:
Every time we make a PR with a functional change to this repo, we create a [release of `awssh`](https://github.com/cisagov/awssh/releases).  As part of that process, we run [`bump_version.sh`](https://github.com/cisagov/awssh/blob/develop/bump_version.sh) to programmatically update the version string of this repo (e.g. `1.1.0` -> `1.2.0`).  Whenever we do that, we would also want to update the version string in `flake.nix` to match the new version.  I'm saying that it would be preferable to modify `bump_version.sh` so that it automatically changes the version in `flake.nix`, rather than someone having to remember to do it manually.

Does that clarify things?

Reply:
@dav3r got it, understood.

Reply:
Looks great now, thanks!